test/e2e/enum.4.ts

```diff
@@ -0,0 +1,4 @@
+export const input = {
+  title: 'Enum with JSON literal',
+  enum: [{type: 'string'}]
```

Comment:
Can you add some more members to this enum? eg.

- Another object
- An array
- A string
- A nested object/array

Reply:
Sure thing @bcherny, I'll do it tomorrow :)